clerk/organizations.go

```diff
@@ -161,3 +161,60 @@ func (s *OrganizationsService) CreateInvitation(params CreateOrganizationInvitat
 	_, err := s.client.Do(req, &organizationInvitation)
 	return &organizationInvitation, err
 }
+
+type ListMembershipsParams struct {
+	OrganizationID string
+	Limit          *int
+	Offset         *int
+}
+
+type publicUserData struct {
+	FirstName       *string `json:"first_name"`
+	LastName        *string `json:"last_name"`
+	ProfileImageURL string  `json:"profile_image_url"`
+	Identifier      string  `json:"identifier"`
+}
+
+type organizationPublicUserData struct {
+	publicUserData
+	UserID string `json:"user_id"`
+}
```

Comment:
:wrench: Why are there two structs instead of just one? What's the purpose of splitting this struct into `publicUserData` and the `UserID`?